d2js/d2wasm/functions.go

```
@@ -241,21 +271,154 @@ func Render(args []js.Value) (interface{}, error) {
 		return nil, &WASMError{Message: "missing 'diagram' field in input JSON", Code: 400}
 	}
 
+	var boardPath []string
+	var noChildren bool
+
+	if input.Opts.Target != nil {
+		switch *input.Opts.Target {
+		case "*":
```

`*` should return an error, since d2js doesn't right now support outputting multiple SVGs.

---

so it does actually support targets with children as it supports animated outputs now. I've updated it a bit so it now throws an error if a target with children is picked but `animateinterval` isn't set. In doing so, I've also updated it so it defaults to only outputting the root board. This _doesn't_ match the CLI (which defaults to '*'), but I think makes more sense here, happy to change if you disagree though